src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -282,6 +286,13 @@ def _gauge_to_dynatrace_metric(self, metric: Metric,
 
     def _histogram_to_dynatrace_metric(self, metric: Metric,
                                        point: HistogramDataPoint):
+        # only allow AggregationTemporality.DELTA
+        if metric.data.aggregation_temporality != AggregationTemporality.DELTA:
+            self._log_temporality_mismatch("Histogram",
+                                           metric,
+                                           AggregationTemporality.DELTA)
```

Comment:
Nit: here you don't specify `supported_temporality` but below you do.

Reply:
Fixed in [dabe990](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/dabe9905bcfdc77cced8aba647032ef43af50f66). :slightly_smiling_face: